webhooks-extension/src/api/index.js

```diff
@@ -16,6 +16,11 @@ import { get, post, deleteRequest } from './comms';
 const apiRoot = getAPIRoot();
 const dashboardAPIRoot = getDashboardAPIRoot();
 
+// Defined here for ease of mocking to test
+export function getSelectedRows(selectedRows) {
+  return selectedRows
+}
```

Comment:
We shouldn't need any additional implementation code relating to testing - I think you could just remove the use of the variable and use the state value directly?

Reply:
Tried my best not to, stuck with it for now unless we use enzyme and mock state atm